exporter/exporter.go

```diff
@@ -527,21 +530,34 @@ func (exporter *NebulaExporter) buildMetricMap(
 
 func convertToMap(metrics []string) map[string]string {
 	matches := make(map[string]string)
-	for t := 0; ; t += 2 {
-		if t+1 >= len(metrics) {
-			break
-		}
-		ok, value := getRValue(metrics[t])
-		if !ok {
-			continue
-		}
-		ok, metric := getRValue(metrics[t+1])
-		if !ok {
-			continue
-		}
-		if value != "" && metric != "" {
-			matches[metric] = value
+	// match metric format
+	// value=0
+	// name=num_queries.rate.60
+	if strings.Contains(metrics[0], "value=") {
+		for t := 0; ; t += 2 {
+			if t+1 >= len(metrics) {
+				break
+			}
+			ok, value := getRValue(metrics[t])
+			if !ok {
+				continue
+			}
+			ok, metric := getRValue(metrics[t+1])
+			if !ok {
+				continue
+			}
+			if value != "" && metric != "" {
+				matches[metric] = value
+			}
 		}
+		return matches
+	}
+
+	// match metric format
+	// slow_query_latency_us.p95.5=0
+	for _, metric := range metrics {
+		s := strings.Split(metric, "=")
+		matches[s[0]] = s[1]
```

Comment:
use getRValue here? 
ok, metric := getRValue(metrics)
if !ok {
  continue
}
matches[s[0]] = metric

Reply:
Right, I will check the length of 's'.